auth/README.md

```diff
@@ -0,0 +1,133 @@
+# Authorization Interceptor
+
+This package provides a server-side gRPC interceptor that interfaces with Themis, a policy engine that is developed and maintained by Infoblox. It is designed to give developers fine-grained control over who (e.g. specific users) or what (e.g. neighboring services) can access their gRPC service's business logic.
+
+## How it Works
+When building a gRPC server, a developer can supply his or her server with a collection of _gRPC server-side interceptors_. Each request that's sent to the gRPC server will traverse the gRPC interceptors before reaching the server's business logic. At any point, an interceptor can stop a given request from advancing to the application.
+
+Here are some common usages for gRPC interceptors.
+
+- Logging (e.g. log to `stdout` on each request)
+- Metrics (e.g. increment a request-counting metric)
+- **Authorization** (e.g. check if the request-sender is allowed to access a given resource)
+
+When the authorization interceptor is enabled, it ensures that whoever or whatever sent the request is allowed to access some endpoint. If they're not, the interceptor issues a `DENY` response, otherwise the request moves to the application itself.
+
+Behold the behavior authorization interceptor as if it were a living, breathing human: _"Hey Themis, is the sender of request XYZ allowed to access endpoint ABC in my service? They're not? Okay, I'll deny the request!"_
+
+## Add Interceptors
+If you already use gRPC with Go, your project probably has a bit of code like this.
+
+```go
+// Define your grpc server
+myServer := grpc.NewServer()
+```
+This is how you would decorate your server with interceptors.
+
+```go
+// Define your grpc server
+myServer := grpc.NewServer(
+  rpc.StreamInterceptor(grpc_middleware.ChainStreamServer(
+    // Define your list of grpc interceptors
+  )),
+)
+```
+
+For more information about adding server-side gRPC interceptors, check out [the documentation](https://github.com/grpc-ecosystem/go-grpc-middleware) on the official gRPC GitHub repository.
```

Comment:
I love your expository, and am happy to see it stay, but in all honesty I think this whole section could have been replaced with just this link :)

Reply:
Okay great! I'm definitely looking to trim this down as much as I can.

Reply:
@markinos Thoughts on this? I added lots of detail because some developers haven't actually used gRPC interceptors before.

Should I just replace this section with a link?

Reply:
My thought is that the intro would just be something like:

This package provides a server-side [gRPC interceptor](https://github.com/grpc-ecosystem/go-grpc-middleware) that interfaces with [Themis](https://github.com/infobloxopen/themis) to give developers fine-grained control over who (e.g. specific users) or what (e.g. neighboring services) can access their gRPC service's business logic.

Reply:
or, even better: 

Behold! This package provides a server-side gRPC interceptor that interfaces with Themis to give developers fine-grained control over who (e.g. specific users) or what (e.g. neighboring services) can access their gRPC service's business logic.